### DISCUSSION TITLE
Problem on running Wireshark

### DISCUSSION THREAD

**Discussion by KrishCrS** [open-quantum-safe/oqs-demos#227] [Q&A] [answered]
I used build.sh to install wireshark, then installed it using "ninja install" inside the wireshark-3.4.9/build directory. Then I tried to run it: sudo wireshark &, but fails to open it. Any suggestions would be greatly appreciated.
![Screenshot from 2023-08-16 17-21-01](https://github.com/open-quantum-safe/oqs-demos/assets/142003662/373ef2e7-10e1-4a3f-b632-20a5c6aa41b6)

**Comment by baentsch** [comment, marked as answer]
This is an XWindow issue in your setup. Please see our documentation and/or use the ready-built wireshark docker image to avoid such issues.

**Reply by baentsch** [reply]
@KrishCrS Has this issue been resolved for you now? Can we close?

**Reply by KrishCrS** [reply]
Yeah, Docker is running fine. It can be closed. Thank you.
